Workshop on EDA.ipynb

```diff
@@ -4,26 +4,55 @@
    "cell_type": "markdown",
```

Comment:
<p>Did you say we will include some of these details and not all? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/niwako/garbanzo/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>